cmd/notify-amqp.go

```diff
@@ -32,6 +32,7 @@ type amqpNotify struct {
 	Exchange     string `json:"exchange"`
 	RoutingKey   string `json:"routingKey"`
 	ExchangeType string `json:"exchangeType"`
+	DeliveryMode uint8  `json:"deliveryMode"`
```

Comment:
It is just using the types in the underlying amqp library. I am not sure wrapping it as a bool (with false for transient and true for persistent) is appropriate because I don't know if this parameter may change in the future. The default value of 0 is implicit and works correctly.

I will add some documentation about this parameter.